src/components/Home/index.js

```diff
@@ -1,27 +1,45 @@
 import React, { Component } from 'react';
 
-import { connect } from 'react-redux'
-import { compose } from 'redux'
-import { firebaseConnect } from 'react-redux-firebase'
-import withAuthorization from '../Auth/withAuthorization'
```

Comment:
Organize imports into two sections: external and internal. Keep each list in alphabetical order by module name.